### ISSUE TITLE
Amidst Icon data

### ISSUE THREAD

**Issue by Zodsmar** [toolbox4minecraft/amidst#425]
I am trying to pull the icon data to find structures based on the icon data, however, if I get back let's say 176, 260. It says a village is there but in amidst it is not there. How is icon data actually being created? I am trying to find a way to find structures with a radius of spawn. 
https://www.youtube.com/watch?v=C817PFudtz0
I made this tool and am trying to update it to have structures also. Or is there a better way to maybe find structures?

**Comment by Zodsmar** [comment]
Anyone?

**Comment by MattSturgeon** [comment]
Well the _icon data_ is [here](https://github.com/toolbox4minecraft/amidst/tree/master/src/main/resources/amidst/icon), but I don't think that's what you're indending to ask...

Based on a [quick search](https://github.com/toolbox4minecraft/amidst/search?q=village&unscoped_q=village), it looks like you are interested in reading [`VillageAlgorithm`](https://github.com/toolbox4minecraft/amidst/blob/edf20f53dd3b9f797be5b25693a6c78e0d6728dc/src/main/java/amidst/mojangapi/world/icon/locationchecker/VillageAlgorithm.java) & [`VillageLocationChecker`](https://github.com/toolbox4minecraft/amidst/blob/b56e31cf8dcb8029ab9ae4248b9703960935d7c1/src/main/java/amidst/mojangapi/world/icon/locationchecker/VillageLocationChecker.java)?

**Comment by Zodsmar** [comment]
@MattSturgeon Yes I am. From testing we haven't be able to figure how to get it to work
